server/webapp/WEB-INF/rails.new/app/assets/stylesheets/new-theme.scss

```diff
@@ -0,0 +1,1356 @@
+    span {
+      line-height:    20px;
+      text-transform: none;
+    }
+  }
+}
+
+.pipelines {
+  .pipeline {
+    padding:       0;
+    border:        0;
+    box-shadow:    none;
+    border-radius: $global-border-radius;
+    box-shadow:    0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+    transition:    all 0.3s cubic-bezier(.25, .8, .25, 1);
+    &:hover {
+      box-shadow: 0 4px 12px rgba(0, 0, 0, 0.25), 0 0px 4px rgba(0, 0, 0, 0.22);
+    }
+  }
+  .pipeline_header {
+    padding:    0px 0 0 15px;
+    background: $pipeline-header-bg;
+    border:     0;
+
+  }
+}
+
+#pipelines .pipelines_selector {
+  position: relative;
+}
+
+.pipeline_header .pipeline_name_link .title.entity_title a {
+  font-weight: 400;
+
+}
+
+.pipeline_bundle div.pipeline .pipeline_instance {
+  padding: 0px 15px;
+  clear:   both;
+}
+
+.pipeline_header h3.entity_title {
+  background-image: none;
+  padding:          10px 10px 5px 0px;
+}
+
+.pipeline_header .pipeline_actions {
+  float:   right;
+  padding: 0;
+  margin:  15px 15px 0 0;
+}
+
+.stage_bar.Passed {
+  background-image: none;
+  background-color: $passed;
+}
+
+.pipeline_bundle .pipeline .status .label {
+  font-size: 12px;
+  margin:    3px 10px 4px 0;
+  a {
+    font-weight: 700;
+  }
+}
+
+.compare_pipeline a {
+  background: url(/go/assets/g9/icons/icon_compare_16.png) transparent no-repeat 0 -1px;
+}
+
+.pipeline .stages .last_run_stage {
+  border-bottom: 6px solid #ccc;
+}
+
+.pipeline_bundle .pipeline_operations {
+  padding: 10px 15px;
+}
+
+.pipeline_bundle .pause_description {
+
+  padding: 0 15px;
+}
+
+.pipelines {
+  .pipeline {
+
+    .unpause_wrapper button.primary {
+      //background-color: $unpause-btn-bg !important;
+
+    }
+    .operate.trigger_wrapper {
+      button.submit {
+        //background: $operate-btn-bg;
+
+      }
+    }
+    .operate.trigger_with_options_wrapper {
+      button.submit {
+        //background: $operate-with-optn-btn-bg;
+
+      }
+    }
+    .operate.pause_wrapper {
+      button.submit {
+        //background: $pause-btn-bg;
+
+      }
+    }
+
+  }
+}
+
+//pipeline history
+
+#bd {
+  border:      0;
+  margin:      0;
+  padding-top: 0;
+  min-width:   auto !important;
+  background:  none;
+}
+
+.container-in-body {
+  padding: 0;
+}
+
+#content_wrapper_outer_pipeline_history #content_wrapper_inner_pipeline_history {
+  padding: 0;
+  margin:  0;
+}
+
+h1#page-title {
+  color:       #333 !important;
+  font-weight: 400 !important;
+}
+
+span.pipeline-buttons a.link_as_button {
+  margin:    5px 0 0 15px;
+  min-width: 40px;
+  &:hover {
+    color: #333;
+  }
+}
+
+.label-filter-group {
+  top:     41px;
+  z-index: 12;
+}
+
+#labelFilterField {
+  border-radius: $global-border-radius;
+
+}
+
+ul.entity_title li {
+  padding-right: 20px;
+  a {
+    font-size: rem-calc(18px);
+  }
+  &.last {
+    > * {
+      font-weight: 500 !important;
+    }
+  }
+}
+
+//css arrows
+
+// Colors
+$border-color: lighten(#000, 75%);
```

Comment:
There's already another `$border-color` defined in `go-variables` lets make sure there's no conflicts in variable names, since variables are global, but defined in individual files?

Is there a naming convention that you think you can introduce to reduce conflicts?